### ISSUE TITLE
CBL 2.0 - Win 7: LiteCore.dll fails to Activate()

### ISSUE THREAD

**Issue by bdgza** [couchbase/couchbase-lite-net#907]
I get this for any (new) Project targetting .NET Framework 4.7 referencing NuGets for Couchbase.Lite and Couchbase.Lite.Support.NetDesktop v2.0.0-db016.

When calling NetDesktop.Activate() on a Windows 7 SP1 64-bit (+all latest important patches), with .NET Framework 4.7, Visual C++ 2015 Update 3 Redistributable, it fails:

> The procedure entry point LoadPackagedLibrary could not be located in the dynamic link library KERNEL32.dll.

After which .NET Framework throws System.BadImageFormatException on LiteCore.dll. This method is only available on Win 8+ and should not be called on Win 7.

> Could not load LiteCore.dll! Nothing is going to work!.

**Comment by borrrden** [comment]
I'm going to have to make a Windows 7 VM to try this out, but I don't see any direct uses of this at all anywhere.  There are some indirect ones but they are only present when compiled using the Windows Store version.  Otherwise, LoadLibraryW is used inside LiteCore, and LoadLibraryEx is used in C# P/Invoke...

**Comment by borrrden** [comment]
I was able to reproduce this, but I still don't know where it is coming from.  It must be a bad build setting somewhere because there is nowhere that I use LoadPackagedLibrary.

**Comment by borrrden** [comment]
Good news, I found the build setting to stop this from happening...and apparently Windows is more proactive about stopping bad DLLs than I thought.  I thought it would only fail if I tried to *use* a function that wasn't allowed, but apparently even compiling it in (even though it is in an unused path on Windows 7) is not allowed and will result in the same error.  I had to change `CopyFile2` to load completely dynamically (via `GetProcAddress`) despite the fact that it seems to be mistakenly marked in the headers as Windows 7+ (with a TODO item present to change that), it is actually Windows 8+, and then fallback to `CopyFileW` on Windows 7.  This change will be in DB018.
